content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -54,3 +65,138 @@ influx write [command]
+  - [from a URL](#write-line-protocol-from-a-url)
+  - [from multiple URLs](#write-line-protocol-from-multiple-urls)
+  - [from multiple sources](#write-line-protocol-from-multiple-sources)
+
+- [Write CSV data](#csv)
+  - [via stdin](#write-annotated-csv-data-via-stdin)
+  - [from a file](#write-annotated-csv-data-from-a-file)
+  - [from multiple files](#write-annotated-csv-data-from-multiple-files)
+  - [from a URL](#write-annotated-csv-data-from-a-url)
+  - [from multiple URLs](#write-annotated-csv-data-from-multiple-urls)
+  - [from multiple sources](#write-annotated-csv-data-from-multiple-sources)
+  - [and prepend annotation headers](#prepend-csv-data-with-annotation-headers)
+
+
+### Line protocol
+
+##### Write line protocol via stdin
+```sh
+influx write --bucket example-bucket "
+m,host=host1 field1=1.2
+m,host=host2 field1=2.4
+m,host=host1 field2=5i
+m,host=host2 field2=3i
+"
+```
+
+##### Write line protocol from a file
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/line-protocol.txt
+```
+
+##### Write line protocol from multiple files
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/line-protocol-1.txt \
+  --file path/to/line-protocol-2.txt
+```
+
+##### Write line protocol from a URL
+```sh
+influx write \
+  --bucket example-bucket \
+  --url https://example.com/line-protocol.txt
+```
+
+##### Write line protocol from multiple URLs
+```sh
+influx write \
+  --bucket example-bucket \
+  --url https://example.com/line-protocol-1.txt \
+  --url https://example.com/line-protocol-2.txt
+```
+
+##### Write line protocol from multiple sources
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/line-protocol-1.txt \
+  --url https://example.com/line-protocol-2.txt
+```
+
+---
+
+### CSV
+
+##### Write annotated CSV data via stdin
+```sh
+influx write \
+  --bucket example-bucket \
+  --format csv \
+  "#datatype measurement,tag,tag,field,field,ignored,time
+m,cpu,host,time_steal,usage_user,nothing,time
+cpu,cpu1,host1,0,2.7,a,1482669077000000000
+cpu,cpu1,host2,0,2.2,b,1482669087000000000
+"
+```
+
+##### Write annotated CSV data from a file
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/data.csv
+```
+
+##### Write annotated CSV data from multiple files
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/data-1.csv \
+  --file path/to/data-2.csv
+```
+
+##### Write annotated CSV data from a URL
+```sh
+influx write \
+  --bucket example-bucket \
+  --url https://example.com/data.csv
+```
+
+##### Write annotated CSV data from multiple URLs
+```sh
+influx write \
+  --bucket example-bucket \
+  --url https://example.com/data-1.csv \
+  --url https://example.com/data-2.csv
+```
+
+##### Write annotated CSV data from multiple sources
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/data-1.csv \
+  --url https://example.com/data-2.csv
+```
+
+##### Prepend CSV data with annotation headers
```

Comment:
Prepend CSV data with annotation headers (and write data from a file?...or can folks do multiple options here?)

Reply:
They can do multiple. This specific example just shows how to do it with a file.